test_scripts/Polices/appID_Management/162_ATF_P_TC_HMI_Status_Appid_Gets_Null_In_Case_Of_PTU.lua

```diff
@@ -77,7 +77,8 @@ end
 commonFunctions:newTestCasesGroup("Test")
 function Test:TestStep_UpdatePolicy()
   testCasesForPolicyAppIdManagament:updatePolicyTable(self, "files/jsons/Policies/appID_Management/ptu_013_2.json")
-  self.mobileSession2:ExpectNotification("OnHMIStatus", { hmiLevel ="NONE", systemContext = "MAIN", audioStreamingState = "NOT_AUDIBLE" })
+  EXPECT_HMINOTIFICATION("SDL.OnAppPermissionChanged", { appID =  data.params.application.appID, appRevoked =  true})
```

Comment:
@pold500, Value of _data.params.application.appID_ will be nil, since _data_ is not defined.
Please use value from self.applications[] table depending on which application is revoked